nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/JoltTransformJSON.java

```diff
@@ -95,6 +99,24 @@
             .required(false)
             .build();
 
+    public static final PropertyDescriptor CUSTOM_CLASS = new PropertyDescriptor.Builder()
+            .name("jolt-custom-class")
+            .displayName("Custom Transformation Class Name")
+            .description("Fully Qualified Class Name for Custom Transformation  Module Directory should be specified")
+            .required(false)
+            .expressionLanguageSupported(false)
```

Comment:
hey @mattyb149 thanks for this review! EL is something I was thinking to do in another iteration; the challenge will be ensuring it will play nicely with the advanced ui. In the advanced UI we have the ability to test out specifications against test data. I think it's important to think through how adding EL will/should reflect there (especially if EL is using flow file attributes which aren't available at that testing point). Also we'll need to ensure that the syntax will play nicely with the codeditor.